cloudFerry.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
```

Comment:
Please add help message to each command available (migrate, evacuate and condense), I guess you can simply copy the docstrings from mentioned commands.